docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -55,9 +55,9 @@ components:
       command: ['sleep', 'infinity']
 ----
 +
-Specify the type of component using the `container` attribute. Use the `image` attribute to specify the image for the component. When defining the `image`, use container naming conventions. For example, the `image` attribute in the preceding example refers to the `docker.io/library/golang:latest` container image.
+Specify the type of component using the `container` attribute. Use the `image` attribute to specify the image for the component. When defining the `image`, use container naming conventions. For example, the `image` attribute in the preceding example refers to the container image,  `docker.io/library/golang:latest`.
 +
-Use the `container` component to augment the image with additional resources and information to allow for integrating the tooling provided by the image with the application that consumes the devfile, for example, {che-prod}.
+Use the `container` component to augment the image with additional resources and information to allow for integrating the tooling provided by the image with the application that consumes the devfile.
```

Comment:
It is hard to unpack the meaning of this sentence. Maybe breaking it into two sentences would help. Make the split around `to allow for`.